bpf/lib/nodeport.h

```diff
@@ -1007,10 +978,10 @@ static __always_inline int nodeport_lb6(struct __ctx_buff *ctx,
 	return DROP_MISSED_TAIL_CALL;
 }
 
-/* See comment in tail_rev_nodeport_lb4(). */
 static __always_inline int rev_nodeport_lb6(struct __ctx_buff *ctx, __u32 *ifindex,
 					    int *ext_err)
 {
+	const bool nat_46x64_fib = nat46x64_cb_route(ctx);
```

Comment:
I'm surprised you're not getting any warnings about `nat_46x64_fib` being potentially unused ...

Reply:
ah nvm, it's always looked at in the FIB path